doc/source/file_formats.rst

```diff
@@ -46,9 +46,48 @@ experimental support for NeuroLucida .asc files.
 SWC
 ---
 
-The SWC format represents a neuron as a tree of 3D points with additional information.
+The SWC format represents a neuron as a set of trees that are connected to a soma.
+The neuronal morphology is encoded as a rooted tree of 3D points and the corresponding radii.
 More information can be found `here <http://research.mssm.edu/cnic/swc.html>`_.
 
+The points of type "1" that represent the soma, have to be in the beginning of the file. 
```

Comment:
We decided earlier that this restriction doesn't apply. Out code doesn't require that any more. See #438 and #478.